# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/uucp/uucp-owner.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./contrib/uupoll.shar.orig	2003-07-09 12:53:50.000000000 -0600
+++ ./contrib/uupoll.shar	2003-07-09 12:54:55.000000000 -0600
@@ -37,7 +37,7 @@
 X
 # The user name/group that should own the resulting executables.
 # Both should run suid.
-owner = uucp.daemon
+owner = uucp.uucp
 X
 # Which mode should the resulting executables have.
 emode = 4111
